extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -642,17 +643,28 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       'gct': this.getLocationQueryParameterValue('google_preview') || null,
       'psts': tokens.length ? tokens : null,
       'gdpr': gdprApplies === true ? '1' : gdprApplies === false ? '0' : null,
-      'gdpr_consent':
-        consentStringType != CONSENT_STRING_TYPE.US_PRIVACY_STRING
-          ? consentString
-          : null,
+      'gdpr_consent': [
+        undefined,
```

Comment:
Is `undefined` needed here? Does it mean no consent strings at all?

Reply:
I was trying to preserve existing behavior of the `consentStringType != CONSENT_STRING_TYPE.US_PRIVACY_STRING` check, and there are existing unit tests of only TC String and no type that defaults to `gdpr_consent`. I think the intended behavior is defaulting to gdpr if a consent string with no type is found.

Reply:
SGTM